documentation/asciidoc/computers/getting-started/installing-from-an-image.adoc

```diff
@@ -1,36 +1,36 @@
 == Installing the Operating System
 
-Raspberry Pi now recommend the use of the https://www.raspberrypi.org/software/[Raspberry Pi Imager] tool to install an operating system on your SD Card. You will need another computer with an SD card reader to install the image.
+Raspberry Pi recommend the use of https://www.raspberrypi.org/software/[Raspberry Pi Imager] to install an operating system on your SD card. You will need another computer with an SD card reader to install the image.
 
-NOTE: Before you start, don't forget to check xref:getting-started.adoc#sd-cards[the SD card requirements].
+NOTE: Before you start, don't forget to check the xref:getting-started.adoc#sd-cards[SD card requirements].
 
-=== Using Raspberry Pi Imager
+IMPORTANT: NOOBS, or New Out Of the Box Software to give it its full name, was an SD card-based installer for Raspberry Pi computers; We no longer recommend using NOOBS.
```

Comment:
Is capital-letter after a semicolon correct?

Reply:
Well spotted - fixed.